lib/postprocessing/image.js

```diff
@@ -459,6 +473,28 @@ async function imagePostProcess(intermediateRendition, rendition, directories) {
     await write(img, rendition.path);
 }
 
+function convertSvg(img, instructions) {
+
+    // Note RSVG is the converter here currently 2020/10/22 causing differences between normal Imagemagic use
+    // Future - allow an instruction to define what color should be converted to transparent
+
+    // ImageMagick automatically keeps aspect ratio
+    // Only using width because ImageMagick will use the smallest value whether it be width or height
+    const width = instructions.width || 400;
+    const fmt = instructions.fmt;
+
+    if (NON_ALPHA_FMT.includes(fmt)) {
+        img.background("rgb(255,255,255)"); // Change transparency to white
+        img.flatten(); // forces the image on its background
+    } else {
+        img.background("rgba(0, 0, 0, 0.0)"); // Keep transparency
+    }
+
+    // some svgs have no size (only percentage width/height), so we scale them to our target size
+    img.in("-size", `${width}`);
```

Comment:
AFAICS [`img.rawSize()`](http://aheckmann.github.io/gm/docs.html#rawSize) maps to the underlying `-size` flag and we should use that. I assume just setting the first argument (width) should work...

Reply:
That is what I had previously but if I use img.rawSize(), this will not be applied at the correct time and the SVG will be upscaled resulting in a fuzzy final rendition.

Reply:
If `rawSize()` doesn't add the `-size` in the right location we shouldn't use it. Probably useful to add a comment regarding this.